README.md

```diff
@@ -122,12 +122,32 @@ More information on the content validation for your custom data types can be fou
 
 ## Data types validation
 
-To validate that the files you have provided have proper json structure, you may execute below script:
+To validate that the schemas you have provided have proper json structure, you may execute below script:
 
 `npm run validate-schema <shema_path>`
 
 where `schema_path` is a full path to the data type json file to validate.
 
+Furthermore, to validate data type values against generated `flat.json` file (or any file with the same structure), following script may be used:
+
+`npm run validate-content <content_validation_cases_path> <schema_path>`
+
+where:
+
+- `content_validation_cases_path` is a full path to json file with validation cases having following structure:
+
+```json
+  [
+    {
+        "type": "absorbed-dose/gy",
+        "content": 45,
+        "expected": "success"
```

Comment:
maybe provide an example for failure as well?

Reply:
Done